### ISSUE TITLE
Unexpected server shut-down when accessing Django-admin

### ISSUE THREAD

**Issue by jwang44** [DDMAL/CantusDB#208]
When running the server locally, I tried to access individual chants in Django-admin (e.g., http://127.0.0.1:3122/admin/main_app/chant/705019/change/). The page wouldn't load, and the server, which was supposed to keep running locally, would stop on its own.

**Comment by jwang44** [comment]
This is not urgent because the user should always use the "real" chant-edit page (e.g., http://127.0.0.1:3122/edit-volpiano/702611?pk=705019) to make changes.
Nonetheless, we should still figure out why this is happening.

**Comment by jacobdgm** [comment]
some notes on this: apparently, a similar issue occurred around when Django 3 was released: Trying to access certain pages in the admin panel would cause the server to crash. Rolling back to 2 was one approach that solved the problem for many people. For others, upgrading their python version did the trick.

Locally, I've tried upgrading Django from 4.0.3 to 4.0.7 . I've also tried upgrading python from 3.8 to 3.10). Neither resolves the issue. (edit: the python upgrade may not have actually been implemented, as I don't understand Docker that much)
